tests/citycoin_test.ts

```diff
@@ -145,6 +146,55 @@ describe('[CityCoin]', () => {
   describe("Read only functions:", () => {
     setupCleanEnv();
 
+    describe("get-coinbase-amount()", () => {
+
+      it("returns correct coinbase amount based on Stacks block height", () => {
+        setupCleanEnv();
+
+        // activate mining
+        chain.mineBlock([
+          client.registerMiner(wallet_3)
+        ]);
+
+        // advance chain to block where mining is active
+        const block = chain.mineEmptyBlock(MINING_ACTIVATION_DELAY);
+        const startHeight = block.block_height - 1;
+
+        const testData = [
+          {blockHeight: startHeight - 1, reward: 0},          // prior to mining activation (no reward)
+          {blockHeight: startHeight, reward: 250000},         // at mining activation (bonus reward)
+          {blockHeight: startHeight + 1, reward: 250000},     // first block after mining activation block (bonus reward)
+          {blockHeight: startHeight + 10000, reward: 250000}, // 1000th block after mining activation block (last bonus reward)
+          {blockHeight: startHeight + 10001, reward: 100000}, // 1001st block after mining activation block (first standard reward)
+          {blockHeight: startHeight + MINING_HALVING_BLOCKS, reward: 100000},              // 1st halving
+          {blockHeight: startHeight + MINING_HALVING_BLOCKS + 1, reward: 50000},           // after 1st halving
+          {blockHeight: startHeight + (MINING_HALVING_BLOCKS * 2), reward: 50000},         // 2nd halving
+          {blockHeight: startHeight + (MINING_HALVING_BLOCKS * 2) + 1, reward: 25000},     // after 2nd halving
+          {blockHeight: startHeight + (MINING_HALVING_BLOCKS * 3), reward: 25000},         // 3rd halving
+          {blockHeight: startHeight + (MINING_HALVING_BLOCKS * 3) + 1, reward: 12500},     // after 3rd halving
+          {blockHeight: startHeight + (MINING_HALVING_BLOCKS * 4), reward: 12500},         // 4th halving
+          {blockHeight: startHeight + (MINING_HALVING_BLOCKS * 4) + 1, reward: 6250},      // after 4th halving
+          {blockHeight: startHeight + (MINING_HALVING_BLOCKS * 5), reward: 6250},          // 5th halving
+          {blockHeight: startHeight + (MINING_HALVING_BLOCKS * 5) + 1, reward: 3125},      // after 5th halving
+          {blockHeight: startHeight + (MINING_HALVING_BLOCKS * 5) + 1234, reward: 3125},   // after 5th halving
```

Comment:
```suggestion
          {blockHeight: startHeight, reward: 250000000000},         // at mining activation (bonus reward)
          {blockHeight: startHeight + 1, reward: 250000000000},     // first block after mining activation block (bonus reward)
          {blockHeight: startHeight + 10000, reward: 250000000000}, // 1000th block after mining activation block (last bonus reward)
          {blockHeight: startHeight + 10001, reward: 100000000000}, // 1001st block after mining activation block (first standard reward)
          {blockHeight: startHeight + MINING_HALVING_BLOCKS, reward: 100000000000},              // 1st halving
          {blockHeight: startHeight + MINING_HALVING_BLOCKS + 1, reward: 50000000000},           // after 1st halving
          {blockHeight: startHeight + (MINING_HALVING_BLOCKS * 2), reward: 50000000000},         // 2nd halving
          {blockHeight: startHeight + (MINING_HALVING_BLOCKS * 2) + 1, reward: 25000000000},     // after 2nd halving
          {blockHeight: startHeight + (MINING_HALVING_BLOCKS * 3), reward: 25000000000},         // 3rd halving
          {blockHeight: startHeight + (MINING_HALVING_BLOCKS * 3) + 1, reward: 12500000000},     // after 3rd halving
          {blockHeight: startHeight + (MINING_HALVING_BLOCKS * 4), reward: 12500000000},         // 4th halving
          {blockHeight: startHeight + (MINING_HALVING_BLOCKS * 4) + 1, reward: 6250000000},      // after 4th halving
          {blockHeight: startHeight + (MINING_HALVING_BLOCKS * 5), reward: 6250000000},          // 5th halving
          {blockHeight: startHeight + (MINING_HALVING_BLOCKS * 5) + 1, reward: 3125000000},      // after 5th halving
          {blockHeight: startHeight + (MINING_HALVING_BLOCKS * 5) + 1234, reward: 3125000000},   // after 5th halving
```

Reply:
Same as the comment above, I don't think this change is necessary.

Reply:
My mistake. It should stay the way it is right now. My suggestions are incorrect.